hadoop-hdfs-project/hadoop-hdfs-httpfs/src/main/java/org/apache/hadoop/fs/http/client/HttpFSFileSystem.java

```diff
@@ -1773,6 +1775,17 @@ public FsStatus getStatus(final Path path) throws IOException {
     return JsonUtilClient.toFsStatus(json);
   }
 
+  public Collection<ErasureCodingPolicyInfo> getAllErasureCodingPolicies() throws IOException {
+    Map<String, String> params = new HashMap<>();
+    params.put(OP_PARAM, Operation.GETECPOLICIES.toString());
+    HttpURLConnection conn =
+        getConnection(Operation.GETECPOLICIES.getMethod(), params, new Path(getUri()
```

Comment:
expend

Reply:
Thanks for your suggestion, I'll make the required code changes promptly